appveyor.yml

```diff
@@ -3,7 +3,6 @@ environment:
   matrix:
     - nodejs_version: "6"
```

Comment:
Travis already tests Node 6 compatibility and the tests that break on windows that don't break on Travis are usually related to the filesystem, which is the same on Node 8.

Removing this would make it as fast as Travis, this way we don't have to wait hours before AppVeyor catches up